packages/configurationmap/README.md

```diff
@@ -0,0 +1,3 @@
+# ConfigurationMap Name
+
+ConfigurationMap documentation
```

Comment:
Much extensive, very communication. ![img](https://i.kym-cdn.com/entries/icons/mobile/000/013/564/doge.jpg) (Seriously, I'm fine with auto generating, but we could give this at least a sentence for what it is for

Reply:
I agree with @DanielMSchmidt , we should have at least a brief description of what the configuration map is and when it is used.

Reply:
Added some documentation.